dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -545,3 +545,81 @@ CWWKD1054.non.unique.result.explanation=The return type of the repository method
 CWWKD1054.non.unique.result.useraction=Update the repository method to either use \
  a return type that supports multiple results or update the repository method \
  to return no more than a single result.
+
+CWWKD1055.unsupported.entity.prop=CWWKD1055E: The {0} property of the {1} entity \
+ has an unsupported type: {2}. Jakarta Data supports Java enumeration types, \
+ the {3} temporal types, and the following basic types: {4}. The built-in \
+ Jakarta Data provider also allows some Jakarta Persistence types, such as \
+ embeddables.
+CWWKD1055.unsupported.entity.prop.explanation=The entity property has a type \
+ that is not supported in Jakarta Data.
+CWWKD1055.unsupported.entity.prop.useraction=Update the entity property to use \
+ one of the supported types.
+
+CWWKD1056.unsupported.keyword=CWWKD1056E: The {0} method of the {1} repository \
+ interface has a name that includes a keyword, {2}, that is not a supported \
```

Comment:
```suggestion
 interface has a name that includes a keyword {2}, that is not a supported \
```

Reply:
I'm fine with removing the first comma, but if we are keeping the remaining comma, then I think we should switch from "that" to "which", otherwise the remaining comma seems out of place.

Reply:
Agree